plugins/extensions/plugins/ZeroEQPlugin.cpp

```diff
@@ -328,6 +380,54 @@ void ZeroEQPlugin::_sceneUpdated( )
             material->setEmission( m.getLightEmission() );
         }
     }
+
+    // Spheres
+    const auto& remoteSpheres = _remoteScene.getSpheresVector();
```

Comment:
when do we start using zerobuf/lexis directly? I'm still surprised you don't get bored yourself always writing this code...